pages/Messages/Validation.md

```diff
@@ -0,0 +1,173 @@
+
+### Content Authenticity
+
+Authentication of a message's contents should be verified by hashing the exact contents of the body returned by the message URI and comparing it with the `contentHash` field of the given message.
+Given that the signature of the message is valid, this should serve as proof that the signing user posted the activity pub content.
+
+## Type Specific Validation
+
+Specific types of DSNP messages will also include their own specific validations depending on the semantic intent of the message type.
+These specific validations follow:
+
+### Broadcasts
+
+1. All broadcast messages with a `"type"` field of `"Follow"` in the activity pub content must include a date in the `"published"` field preceded by a follow [Graph Change](/Messages/Overview#GraphChange) DSNP message with no unfollow events between the time of the follow and the broadcast messages.
+1. All broadcast messages with a `"type"` field of `"Like"` or `"Dislike"` in their activity pub content must be preceded by a [Reaction](/Messages/Overview#Reaction) DSNP message referring to the same object, i.e. that the `URI` message field of DSNP message identified by the `inReplyTo` field of the reaction is identical to the `"object"` field of the like or dislike activity pub content.
+1. All broadcast messages with a `"type"` field of `"Undo"` must include an `"object"` field with either a valid DSNP identifier referring to a previously announced broadcast message or a URI matching the URI of a previously announced broadcast event.
+
+### Replies & Reactions
+
+1. All reply and reaction messages must include an `inReplyTo` field identifying an existing DSNP message.
+1. All reply and reaction messages must be from and signed by a user, the replier, for which the user who posted the original content referred to in the `inReplyTo` field has never previously published a broadcast message with a activity pub content `"type"` field of `"Block"` and `"object"` field containing the string `dsnp://` followed by the user id of the replier, thereby indicating that the original poster has previously blocked the replier.
+
+## Optional Validations
+
+In addition to the aforementioned general validations on all messages and type specific validation on particular messages, there are also a set of optional validations that will be defined below and implementers are encouraged to support, however users, archivists and indexers may choose to ignore depending on their legal jurisdiction and individual preference.
+As such, it is strongly encouraged to make these features configurable by the end user in all implementations as allowable by law.
+
+### Blocklists & Allowlists
+
+Users may maintain private lists of other users who they do not wish to publicly block but choose to ignore all messages from.
+Likewise, archivists and indexers may also choose to ignore messages from users with repeated patterns of malicious or other objectionable behavior.
```

Comment:
This could also have legal ramifications. Maybe it could be considered as destroying evidence, but I don't know.  This also seems like something we should run by a legal department.  This line of thought only occurred to me as I started reading through this PR.

Reply:
Absolutely, we should run this by legal. For what it's worth, I meant to say announcer here instead of archivist, but I supposed the same concerns apply.

Reply:
Switching it to announcer clears all of my concerns. No one is required to announce content they disagree with (in line with no one is required to host your website), but you can do it yourself. Archivists have (potentially) a much more neutral role to play.